javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentClassLoader.java

```diff
@@ -63,15 +66,23 @@ public class AgentClassLoader extends URLClassLoader {
   private final URL jarBase;
   private final String jarEntryPrefix;
   private final CodeSource codeSource;
+  private final boolean isSecurityManagerSupportEnabled;
   private final Manifest manifest;
 
+  public AgentClassLoader(File javaagentFile, String internalJarFileName) {
```

Comment:
Nit: do we actually need this constructor? Can't we just inline this?

Reply:
It is used in tests that just want to construct a new `AgentClassLoader` using just the first argument. WDYT if we keep this constructor just for tests and remove `internalJarFileName` argument because tests always pass `""` for it?

Reply:
Sounds good 👍 
Can you just add some `// used in tests` comment?

Reply:
Comment added